gpkit/nomials/nomial_math.py

```diff
@@ -494,7 +491,7 @@ def _gen_unsubbed(self):
                              "be converted to units of %s" %
                              (self.p_lt, self.m_gt))
 
-        p.exps, p.cs = self._simplify_posy_ineq(p.exps, p.cs)
```

Comment:
Not clear to me why we should remove simplification. Seems like this was a mapping bug, not a simplification bug, right? Isn't this only an indirect fix?